packages/office-ui-fabric-react/src/components/Breadcrumb/Breadcrumb.styles.ts

```diff
@@ -0,0 +1,187 @@
+import {
+  HighContrastSelector,
+  IRawStyle,
+  IStyle,
+  ITheme,
+  ScreenWidthMaxMedium,
+  ScreenWidthMaxSmall,
+  ScreenWidthMinMedium,
+  getFocusStyle,
+  getScreenSelector
+} from '../../Styling';
+
+export interface IBreadcrumbStyleProps {
+  className?: string;
+  theme: ITheme;
+}
+export interface IBreadcrumbStyles {
+  root: IStyle;
+  list: IStyle;
+  listItem: IStyle;
+  chevron: IStyle;
+  overflow: IStyle;
+  overflowButton: IStyle;
+  itemLink: IStyle;
+  item: IStyle;
+}
+
+const SingleLineTextStyle: IRawStyle = {
+  whiteSpace: 'nowrap',
+  textOverflow: 'ellipsis',
+  overflow: 'hidden'
+};
+
+const MinimumScreenSelector = getScreenSelector(0, ScreenWidthMaxSmall);
+const MediumScreenSelector = getScreenSelector(ScreenWidthMinMedium, ScreenWidthMaxMedium);
```

Comment:
I know this was brought over from old code but I wonder if this should be related parent container size instead of the whole screen? I think it's making an assumption that the parent container IS always the screen.. when I see components referring to any screen dimensions, it seems there's a chance that they may not truly be hierarchical...

Reply:
@JasonGore not really possible with media queries; you need element queries, but this isn't exactly possible without some hacks at this point. (AFAIK)

I had refactored this in my 6.0 changes to use resizegroup, which abstracts measuring. But i'm going to back out those changes to accelerate 6.0 updates.